pkg/cluster/kubeadm/config.go

```diff
@@ -61,6 +61,15 @@ clusterName: {{.ClusterName}}
 # to the cluster after rewriting the kubeconfig to point to localhost
 apiServerCertSANs: [localhost]
 kubernetesVersion: {{.KubernetesVersion}}
+# TODO(bentheelder): fix this upstream!
+# we need nsswitch.conf so we use /etc/hosts
+# https://github.com/kubernetes/kubernetes/issues/69195
```

Comment:
once this is fixed upstream we should clean this up and try to only do it when necessary or just remove it. I'll work on an upstream fix tomorrow